src/main/java/com/iotiq/application/messages/product/ProductFilter.java

```diff
@@ -111,7 +163,7 @@ public Specification<Product> buildSpecification() {
                 .and(isIn("ageGroup", "adultAgeGroup", adultAgeGroups))
                 .and(isIn("ageGroup", "childrenAgeGroup", childrenAgeGroups))
                 .and(isIn("applicationAreaGroup", "applicationArea", applicationAreas))
-                .and(isIn("applicationAreaGroup", "frequency", frequencies))
+                .and(isIn("applicationAreaGroup", "usageCycle", usageCycles))
```

Comment:
Why aren't we using meta models? @oguzIotiq

Reply:
@ozdkaan can you open an issue for that?

Reply:
https://github.com/Iotiq-Erste/kimono-application/issues/53